README_CN.md

```
@@ -0,0 +1,475 @@
+<h1 align="center">Voicebox</h1>
+
+<p align="center">
+  <strong>开源 AI 语音工作室</strong><br/>
+  本地优先的语音克隆、语音生成、全局听写、AI 代理语音输出
+</p>
+
+---
+
+## 📋 目录
+
+- [环境要求](#环境要求)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)